modin/data_management/data_manager.py

```diff
@@ -1792,17 +1794,23 @@ def tail(self, n):
             DataManager containing the last n rows of the original DataManager.
         """
         # See head for an explanation of the transposed behavior
+        if n < 0:
+            n = max(0, len(self.index) + n)
+        if n == 0:
+            index = self.index[:0]
```

Comment:
shouldn't this be `pandas.Index([])`? This can return a `RangeIndex` where `start` and `stop` are 0, which isn't what pandas does.